Source/Models/Packages/Package/Package.mm

```diff
@@ -898,6 +898,27 @@ - (BOOL) matches:(NSArray *)query {
     NSString *string;
     NSRange range;
     NSUInteger length;
+	
+	// If we are filtering for an author
+	if ([[(NSString*)query[0] lowercaseString] isEqualToString:@"author:"]) {
+		NSMutableArray *authorTerms = [query mutableCopy];
+		[authorTerms removeObjectAtIndex:0];
+		[self parse];
+		if ([self author] && [self author].name && ![[self author].name isEqualToString:@""]) {
```

Comment:
In the future instead of `![[self author].name isEqualToString:@""]` you could do `[self author].length > 0`